File: app/src/main/java/com/shevart/androidcorelearn/common/ShortTextWatcher.java
package com.shevart.androidcorelearn.common;

import android.text.TextWatcher;

/***
 *  This class hide two methods which I use seldom. Make code shorter
 */
public abstract class ShortTextWatcher implements TextWatcher {
    @Override
    public void beforeTextChanged(CharSequence s, int start, int count, int after) {

    }

    @Override
    public void onTextChanged(CharSequence s, int start, int before, int count) {

    }
}

File: app/src/main/java/com/shevart/androidcorelearn/common/ShortTextWatcher.kt
package com.shevart.androidcorelearn.common

import android.text.TextWatcher

/***
 * This class hide two methods which I use seldom. Make code shorter
 */
abstract class ShortTextWatcher : TextWatcher {
    override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {

    }

    override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {

    }
}